1. Select basics/Task 1 Select/Result/Task1_Natalli_Sholomitskaya.sql

```diff
@@ -0,0 +1,29 @@
+select * from employees;
+
+select department_id, department_name, manager_id as "MNG"
+from departments;
+
+select employee_id, first_name, last_name, phone_number, department_id
+from employees;
+
```

Comment:
Like here you can add record 
/* Task 4
Create a query to display the first name, last name, hire date, salary, and salary after a raise of 20%. Name the last column (salary after a raise) heading as “ANNUAL_SAL” */